### PR TITLE
Add Esperanto translation

### PR DESCRIPTION
**Pull request by Foorack**
# Translation Info
**Important to note: I have not studied Esperanto for very long. This translation is most likely not error-free.**

I have asked other Esperanto speakers if they can quickly have a look at it. For anyone who hasn't used Github before, click on "Files changed" in the middle of the screen to see the difference. Optionally you can also click on the links in the table below.

The following things should not be translated:

- Minecraft commands such as `/changepassword`, `/unregister` and `/email change`. Those are hard coded and cannot be changed.
- Variables such as %count, %valid and REG_EX.
- The color codes should not be changed e.g. `&c` or `&4.

# Files
### help_eo.yml
| Language      | Link          |
| ------------- | ------------- |
| English (Original)      | [src/main/resources/messages/help_en.yml](https://github.com/Foorack/AuthMeReloaded/blob/master/src/main/resources/messages/help_en.yml) |
| Esperanto      | [src/main/resources/messages/help_eo.yml](https://github.com/Foorack/AuthMeReloaded/blob/master/src/main/resources/messages/help_eo.yml)      |
### messages_eo.yml
| Language      | Link          |
| ------------- | ------------- |
| English (Original)      | [src/main/resources/messages/messages_en.yml](https://github.com/Foorack/AuthMeReloaded/blob/master/src/main/resources/messages/messages_en.yml) |
| Esperanto      | [src/main/resources/messages/messages_eo.yml](https://github.com/Foorack/AuthMeReloaded/blob/master/src/main/resources/messages/messages_eo.yml)      |

### Please do not merge until the quality has been checked by someone who can fully speak Esperanto.
Constructive feedback is appreciated. :smiley:

### CODE REVIEW

File: src/main/resources/messages/help_eo.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+common:
+    header: '==========[ AuthMeReloaded Helpo ]=========='
+    optional: 'Laŭvola'
+    hasPermission: 'Vi havas permeson'
+    noPermission: 'Neniu permeson'
+    default: 'Defaŭlta'
```
Comment (Fer22f) [inline src/main/resources/messages/help_eo.yml]: *apriora*, ne *defaŭlta*

File: src/main/resources/messages/help_eo.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+common:
+    header: '==========[ AuthMeReloaded Helpo ]=========='
+    optional: 'Laŭvola'
+    hasPermission: 'Vi havas permeson'
+    noPermission: 'Neniu permeson'
```
Comment (Fer22f) [inline src/main/resources/messages/help_eo.yml]: Se vi volas meti la akuzativon ĉi tie, bonvolu meti akuzativon ankaŭ al *neniu*.
Kiel "neniun permeson" (vi havas neniun permeson)

File: src/main/resources/messages/help_eo.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    default: 'Defaŭlta'
+    result: 'Rezulto'
+    defaultPermissions:
+        notAllowed: 'Neniu permeson'
```
Comment (Fer22f) [inline src/main/resources/messages/help_eo.yml]: Sama kiel la kvina linio

File: src/main/resources/messages/help_eo.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        opOnly: 'Nur OP''s'
```
Comment (Fer22f) [inline src/main/resources/messages/help_eo.yml]: Esperanto ne havas la saksan genitivon. Proponita traduko: *nur el OP*

File: src/main/resources/messages/help_eo.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    alternatives: 'Alternativoj'
+    children: 'Komandoj'
+
+commands:
+    authme.register:
+        description: 'Registri ludanto'
+        detailedDescription: 'Registri la specifita ludanto kun la specifita pasvorton.'
+        arg1:
+            label: 'ludanto'
+            description: 'Ludanto nomo'
```
Comment (Fer22f) [inline src/main/resources/messages/help_eo.yml]: Bonvolu ne havi du o-vortoj apud la alia! Proponita traduko: *ludanta nomo*

File: src/main/resources/messages/messages_eo.yml [hunk 6]
```diff
@@ -0,0 +1,106 @@
+# Registration
+reg_msg: '&3Bonvolu registri al la servilo per la komando: /register <pasvorto> <konfirmiRetpoŝto>'
+usage_reg: '&cUzado: /register <pasvorto> <konfirmiRetpoŝto>'
+reg_only: '&4Nur registritaj uzantoj povas aliĝi la servilo! Bonvolu viziti http://example.com registri vin mem!'
```
Comment (Fer22f) [inline src/main/resources/messages/messages_eo.yml]: *la servilo**n***.

File: src/main/resources/messages/messages_eo.yml [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+registered: '&2Sukcese registris!'
+reg_disabled: '&cEn-ludo registriĝo estas malebligita!'
+user_regged: '&cVi jam registris tiun uzantnomon!'
+
+# Password errors on registration
+password_error: '&cLa pasvortoj ne kongruas, kontrolu ilin!'
+password_error_nick: '&cVi ne povas uzi vian nomon kiel pasvorto, bonvolu elekti alian...'
+password_error_unsafe: '&cLa elektita pasvorto estas danĝere, bonvolu elekti alian...'
+password_error_chars: '&4Via pasvorto enhavas kontraŭleĝan karakteroj. Permesita signoj: REG_EX'
+pass_len: '&cVia pasvorto estas tro mallonga aŭ tro longa! Bonvolu provi per alia!'
```
Comment (Fer22f) [inline src/main/resources/messages/messages_eo.yml]: *provi per alia*?

File: src/main/resources/messages/messages_eo.yml [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+# Login
+usage_log: '&cUzado: /login <pasvorto>'
+wrong_pwd: '&cErara pasvorto!'
+login: '&2Sukcesaj ensaluto!'
```
Comment (Fer22f) [inline src/main/resources/messages/messages_eo.yml]: La pluralo ne necesas... ĉu ne?

**Comment by Foorack** [comment]
### Ending review session
Thanks everyone for the feedback! :smile: :star: 

@ljacq This should now be at a high enough level for inclusion. At least I personally cannot find any further errors.

Ready to merge. :+1:

**Comment by Foorack** [comment]
# Ending review session
About 3 days have passed since the creation of this merge request. This should now be at a high enough level for inclusion. Thanks to the people who have provided feedback! :star: 

---

@ljacqu Should be ready to merge. :+1:

**Comment by Xephi** [comment]
Thanks for your work :)

**Comment by ljacqu** [comment]
Thanks a lot @Foorack ! :)

**Pull request merged**